test/functional/Java19andUp/build.xml

```diff
@@ -49,22 +48,16 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<echo>===debug:                        on</echo>
 		<echo>===destdir:                      ${DEST}</echo>
 
-		<if>
-			<equals arg1="${JDK_VERSION}" arg2="19"/>
-			<then>
-				<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1">
-					<src path="${src}" />
-					<src path="${TestUtilities}" />
-					<compilerarg line='--enable-preview --source ${JDK_VERSION}' />
-					<classpath>
-						<pathelement location="${LIB_DIR}/testng.jar" />
-						<pathelement location="${LIB_DIR}/jcommander.jar" />
-						<pathelement location="${LIB_DIR}/asm.jar" />
-						<pathelement location="${build}" />
-					</classpath>
-				</javac>
-			</then>
-		</if>
+		<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1">
+			<src path="${src}" />
+			<compilerarg line='--enable-preview --source ${JDK_VERSION}' />
+			<classpath>
+				<pathelement location="${LIB_DIR}/testng.jar" />
+				<pathelement location="${LIB_DIR}/jcommander.jar" />
+				<pathelement location="${LIB_DIR}/asm.jar" />
+				<pathelement location="${build}" />
+			</classpath>
+		</javac>
```

Comment:
There is a merge conflict here against PR #15767.

Reply:
Resolved the conflict in the building script.